src/main/java/com/acwong/peelplaylistcheck/model/Content.java

```diff
@@ -0,0 +1,61 @@
+package com.acwong.peelplaylistcheck.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown=true)
+public class Content extends VideoGroup {
```

Comment:
Should this class extend Video or VideoGroup?

Reply:
This extends VideoGroup.